Register country

mattRedBox opened the pull request:
Initial commit for country registration, on behalf of ODI QLD, Aust.
TODO : help text, redirect to register country on login (for current users), and statistics - will follow with these soon.

Stephen-Gates commented:
This is for work described in the [country-level statistics](https://trello.com/c/jJBvlcW5/30-country-level-statistics) on the Labs Trello board . Spec and other resources available there. Thanks Matt.

mattRedBox commented:
Added instruction/help text.
Added redirect to registration page if no country on registration/login of existing user.

Hey guys, please let me know if more needed (e.g., test coverage, adherence to code style)

(Following requirements from: https://trello.com/c/jJBvlcW5/30-country-level-statistics)
To follow: statistics tasks.
